GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryHS.cfg

```diff
@@ -0,0 +1,71 @@
+PART
+{
+	name = ROC-MercuryHS
+	module = Part
+	author = CobaltWolf, capkirk
+
+	RSSROConfig = true
+
+	MODEL
+	{
+		model = ROCapsules/Assets/BDB/Mercury/bluedog_Mercury_Heatshield
+		rotation = 0, 180, 0
+	}
+	
+	rescaleFactor = 1.512
+	node_stack_bottom = 0.0, -0.0820906, 0.0, 0.0, -1.0, 0.0, 1
+	node_stack_top = 0.0, 0.037256, 0.0, 0.0, 1.0, 0.0, 1
+	CoPOffset = 0.0, 1.0, 0.0
+	CoLOffset = 0.0, -0.15, 0.0
+	//buoyancyUseCubeNamed = Default
+	buoyancy = 0.1
+	buoyancyUseSine = False
+	sound_vent_large = decouple
+
+	TechRequired = engineering101
+	entryCost = 600
+	cost = 300
+	category = Thermal
+	subcategory = 0
+	title = Mercury Heatshield
+	manufacturer = McDonnell Aircraft
+	description = Small 1.89m heatshield for the Mercury capsule. Place underneath, before adding the retropack. Does not include a fairing. Comes with a built in airbag for a nice comfy landing.
+	attachRules = 1,0,1,0,0
+	mass = 0.05625
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 200
+	breakingTorque = 200
+	crashTolerance = 9
+	maxTemp = 3300
+	fuelCrossFeed = False
+	PhysicsSignificance = 0
+	stageOffset = -1
+	childStageOffset = 1
+	thermalMassModifier = 1.0
+	bulkheadProfiles = size1
+	stagingIcon = WHEEL //DECOUPLER_VERT
+
+	tags = mercury hermes heatshield 1.25
+
+	MODULE
+	{
+		name = ModuleAnimateGeneric
+		animationName = deploy
+		isOneShot = false
+		startEventGUIName = Deploy Landing Bag
+		endEventGUIName = Retract Landing Bag
+		actionGUIName = Toggle Landing Bag
+		allowAnimationWhileShielded = False
+		restrictedNode = bottom
+		disableAfterPlaying = true
+	}
+
+	//use new heat shield tagging system to configure the shield instead
+	resetHeatShieldMass = True
+	resetHeatShieldAblator = True
+	heatShieldTag = LEO
+	heatShieldDiameter = 1.89
```

Comment:
Note that using these tags means RO will reset the mass and the maxTemp.  So declaring those fields in this PART config is not strictly necessary and may be misleading.  Not an issue for merging this, we've only just gone through the round of standardizing what engine configs look like, we haven't made it to the part definitions there, let alone for heatshields.